feed.go

```diff
@@ -49,6 +68,7 @@ type Feed struct {
 	Items       []*Item
 	Copyright   string
 	Image       *Image
+	Options     *SerializeOpts
```

Comment:
This shouldn't be by pointer. If you have multiple `Feed` objects and they all get passed the same `Options` and then one of them mutates it later, it will affect all the objects.